packages/moonstone/Panels/Header.module.less

```diff
@@ -155,6 +155,11 @@
 		.padding-start-end(0, var(--moon-panels-controls-width));
 	}
 
+	.header.centered.standard .title,
+	.header.centered.standard .headerInput {
+		.padding-start-end(var(--moon-panels-controls-width), var(--moon-panels-controls-width));
```

Comment:
We're incurring a bit of overhead by using the mixin when both values are the same since it will produce extra selectors for rtl with the same rules.

@teckliew + @Djspaceg - worth revising?

Reply:
The use of the mixin, while seemingly verbose, allows this rule to overcome the rule on line 155. If we simply applied `padding` rules here, we would not be targeting the higher precedence RTL rules established on line 155 above.

If we instead used a newer (less supported) rule like `padding-block-start` and `padding-block-end`, we wouldn't need to use the mixin, and could use a simpler override on line 160.

Reply:
:+1: Thanks for the explanation.